_includes/specials/taproot/en/13-signet.md

```diff
@@ -0,0 +1,132 @@
+Although you [can't safely use taproot before block
+{{site.trb}}][taproot safety] on mainnet, you can use taproot today with
+either testnet or signet.  Compared to a creating a local test network
+with Bitcoin Core's regtest mode, as done in the [Optech taproot
+workbooks][], using testnet or signet makes it easier to test how your
```

Comment:
Can link to signet topic listing here or elsewhere in the special

Reply:
It's linked to below, on line 9.